README.md

```diff
@@ -1,2 +1,22 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) The algorithm checks if there are duplicate values in a list, returning true if there are.  
+
+b) If there are no duplicates, the algorithm will have to iterate the entire length of the list for each item in the list, resulting in the number of passes being (length of list ^ 2).   
+
+c) The function now checks for duplicates before the current item in the list, rather than checking the whole list. However, when the last item in the list is reached, every item before it will be checked, therefore any duplicates will still be found.  
+
+d) As only the items before the current item are checked, there are approximatly half the number of comparisons to make.  
```

Comment:
✔️ since the original algorithm compares each pair of elements twice: A==B and B==A